### PR TITLE
locheck: add support for linux.

### PR DESCRIPTION
**Pull request by AnessZurba**
Hi,

Full disclaimer: I'm no expert in running swift on Linux.

This is a try to make this project compatible with linux, the changes are very minimal, as the code doesn't use anything platform-specific (from what I can see)

Replace Darwin imports with Glibc, and a small change in Makefile to make it multi-platform

I did try to compile, install and run on ubuntu on a real iOS project with lots of localizations, it seems to work. (Although the version I ran was by replacing Darwin by Glibc, not the one with `if`s)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ VERSION = 0.9.11
 
 PREFIX = /usr/local
 INSTALL_PATH = $(PREFIX)/bin/$(EXECUTABLE_NAME)
-BUILD_PATH = .build/apple/Products/Release/$(EXECUTABLE_NAME)
+BUILD_PATH = .build/release/$(EXECUTABLE_NAME)
```
Comment (stevelandeyasana) [inline Makefile:7]: Can you confirm that this doesn't break the Mac build?

**Review by stevelandeyasana — COMMENTED** [review]
Wow, thanks! Looking forward to getting this landed and expanding our support.

File: Makefile (hunk 1 above)
Comment (AnessZurba) [reply Makefile]: Actually it doesn't, fixed now

**Review by stevelandeyasana — APPROVED** [review]
Thanks! I'll consider doing a release soon.

**Pull request merged**